miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -4600,6 +4600,10 @@ public static void setBoolean(@Nonnull Consumer<Boolean> setter, Boolean value,
     }
   }
 
+  public static void setInteger(@Nonnull Consumer<Integer> setter, String value) {
+    setter.accept(isStringEmptyOrNull(value) ? null : Integer.valueOf(value));
+  }
```

Comment:
There's already a `setInteger` method that can be used; it has an extra parameter to specify whether null is allowed (should be `true` in this case)